### PR TITLE
Bootstrap everything, and start prettification

### PR DESCRIPTION
**Pull request by DJRHails**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a feature/bugfix branch** (right side). Don't request from master!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### Description
Converts everything to use bootstrap over material ui. Adds some basic UI improvements for registration and cards for Governance.

### Checklist
* [x] Have you done your changes on a separate branch. Branches MUST have descriptive names that start with either the `fix/` or `feature/` prefixes. Good examples are: `fix/signin-loop` or `feature/pr-templates`.
* [x] Have you got descriptive commit messages that would make sense if prefixed with "This commit will ..."
* [x] Will you squash when you merge this PR?

💔Thank you!

### CODE REVIEW

File: app/src/components/ArtworkItem.tsx [hunk 1]
```diff
@@ -14,10 +14,12 @@ type ArtworkItemState = {
 
 class ArtworkItem extends React.Component<ArtworkItemProps, ArtworkItemState> {
   componentDidMount (): void {
-    this.props.drizzle.contracts.ArtifactRegistry.methods.tokenOfOwnerByIndex(
+    const registry = this.props.drizzle.contracts.ArtifactRegistry;
+    console.log('registry');
```
Comment (JamesDalboth) [inline app/src/components/ArtworkItem.tsx]: Leftover debug message?

File: app/src/components/Register.tsx [hunk 2]
```diff
@@ -98,167 +116,164 @@ class Register extends React.Component<RegisterProps, RegisterState> {
...
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
+          </Form.Group>
+          <Form.Group as={Col} controlId="artistBirthYear">
+            <Form.Label>Artist Birth Year</Form.Label>
+            <Form.Control
+              required
+              type="text"
+              onChange={this.inputChangeHandler}/>
+            <Form.Control.Feedback>Looks good!</Form.Control.Feedback>
```
Comment (JamesDalboth) [inline app/src/components/Register.tsx]: I'm seeing "Looks good!" alot, maybe worth a variable, maybe not

**Review by JamesDalboth — APPROVED** [review]
Small comments, up to you to fix, don't think needs reapproval after though

File: app/src/components/ArtworkItem.tsx (hunk 1 above)
Comment (DJRHails) [reply app/src/components/ArtworkItem.tsx]: Definitely

File: app/src/components/Register.tsx (hunk 2 above)
Comment (DJRHails) [reply app/src/components/Register.tsx]: Probably, got a tad lazy with my feedback stuff.

**Review by matt-malarkey — APPROVED** [review]

**Pull request merged**